Sources/TokamakLint/Helpers/GetComponents.swift

```diff
@@ -1,27 +1,30 @@
 //
-//  GetHookedComponents.swift
+//  GetComponents.swift
 //  TokamakLint
 //
-//  Created by Matvii Hodovaniuk on 5/23/19.
+//  Created by Matvii Hodovaniuk on 5/28/19.
 //
 
 import SwiftSyntax
 
+let hookedComponentProtocols = ["CompositeComponent", "LeafComponent"]
+let componentsProtocols = ["CompositeComponent", "LeafComponent",
```

Comment:
please rename this to `componentProtocols` for consistency, you can also reuse `hookedComponentProtocols` here via array concatenation